Flyer Basics.ipynb

```diff
@@ -750,9 +767,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.7.7 64-bit ('bluesky_2020_5': conda)",
```

Comment:
will change this be a problem?

Reply:
The change in `display_name` isn't really a problem (at most a cosmetic inconsitency) but the change in the kernel `name` below causes Ci to fail with

```
jupyter_client.kernelspec.NoSuchKernel: No such kernel named python37764bitbluesky20205conda414c31c00fba48028d4852f6340d7af7
```

and would cause the same issue locally. I'll push a commit patching it.